actionlist/package.json

```diff
@@ -5,15 +5,19 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest",
-    "lint": "./node_modules/eslint/bin/eslint.js *.js **/*.js --ignore-pattern node_modules/"
+    "lint": "./node_modules/eslint/bin/eslint.js . --ignore-pattern node_modules/"
   },
   "dependencies": {
     "@fugood/react-native-call-modal": "0.0.6",
     "react": "15.4.2",
     "react-native": "0.40.0",
-    "react-native-button": "^1.8.2"
+    "react-native-button": "^1.8.2",
+    "react-redux": "^5.0.3",
```

Comment:
You shouldn't have react-redux as both a production and dev dependency. Why is it needed as a production dependency? Babel should transpile it away when you eject the project.